### PR TITLE
fix: month/year panel of DatePicker[showToday=false], close: #5620

### PR DESCRIPTION
**Pull request by benjycui**
Close: https://github.com/ant-design/ant-design/issues/5620

### CODE REVIEW

File: components/date-picker/style/MonthPanel.less [hunk 1]
```diff
@@ -18,10 +22,14 @@
   .calendarPanelHeader(~"@{calendar-prefix-cls}-month-panel");
 }
 
+.@{calendar-prefix-cls}-month-panel-body {
+  height: ~"calc(100% - 34px)";
```
Comment (afc163) [inline components/date-picker/style/MonthPanel.less]: Consider compatibility.

Comment (benjycui) [inline components/date-picker/style/MonthPanel.less]: ref: http://caniuse.com/#search=calc

Comment (afc163) [inline components/date-picker/style/MonthPanel.less]: What about IE9/10

Comment (benjycui) [inline components/date-picker/style/MonthPanel.less]: It works fine in IE9

image

**Pull request merged**